lms/djangoapps/experiments/utils.py

```diff
@@ -313,7 +313,7 @@ def get_base_experiment_metadata_context(course, user, enrollment, user_enrollme
         enrollment_time = enrollment.created
 
         try:
-            schedule_start = enrollment.schedule.start
+            schedule_start = enrollment.schedule.start_date
```

Comment:
If in the last PR (4/5) you removed some writes to the old field, how can this PR be changing any business logic?  Won't that break the logic between 4 and 5?

Reply:
In step 4/5, not all writes could be removed from old field `start` as it non-NULL non-blank field. All writes from old field could be removed only while this field is removed. 
I've not changed any business logic. Can you please elaborate if you think there's mismatch/ change in business login in 4/5 & 5/5

Reply:
@pwnage101 I was also thinking about moving `remove old field` migration to separate PR from here so that it'll be easier in case if it needs to be reverted for any reason. Thoughts!